src/ontology/mondo-edit.obo

```diff
@@ -155715,7 +155715,7 @@ relationship: disease_has_feature MONDO:0015791 {source="Orphanet:178345"} ! per
 relationship: disease_has_location UBERON:0000990 ! reproductive system
 relationship: excluded_subClassOf MONDO:0016072 {source="Orphanet:178345", source="https://orcid.org/0000-0001-5208-3432"} ! obsolete anomaly of puberty or/and menstrual cycle of genetic origin
 relationship: excluded_subClassOf MONDO:0018561 {source="Orphanet:178345", source="https://github.com/Orphanet/ORDO/issues/19", source="https://orcid.org/0000-0001-5208-3432"} ! precocious puberty in female
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2594 {source="MONDO:mim2gene_medgen", source="OMIM:139300"} ! CYP19A1
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2594 {source="MONDO:mim2gene_medgen"} ! CYP19A1
```

Comment:
This is correct according to the OMIM record. 
We should go back and remove the whole annotation.